src/ILCompiler.RyuJit/src/JitInterface/CorInfoImpl.RyuJit.cs

```diff
@@ -340,9 +294,15 @@ private ISymbolNode GetHelperFtnUncached(CorInfoHelpFunc ftnNum)
                 case CorInfoHelpFunc.CORINFO_HELP_NEWFAST:
                     id = ReadyToRunHelper.NewObject;
                     break;
+                case CorInfoHelpFunc.CORINFO_HELP_NEWSFAST:
+                    return _compilation.NodeFactory.ExternSymbol("RhpNewFast");
```

Comment:
Our normal pattern is to assign a `ReadyToRun` ID for these and pipe it through places, but it seem like we can do shortcuts like this. I can make this go the full ID route instead...

Reply:
I agree that the shortcuts are better. Going via `ReadToRun` IDs for fragile images does not make a lot of sense.